index.js

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable max-classes-per-file */
+import ListBooks from './modules/listBookClass.js';
+
```

Comment:
- [Optional] Your nav is not functioning currently, kindly import your nav module to the index.js file to fix this. Add `import './modules/nav.js';` in index.js.